src/sas/sasview/local_config.py

```diff
@@ -46,12 +48,20 @@
 _acknowledgement_preamble_bullet4 =\
 '''Send us your reference for our records: developers@sasview.org'''
 _acknowledgement_publications = \
-'''This work benefited from the use of the SasView application, originally developed under NSF Award DMR-0520547. SasView also contains code developed with funding from the EU Horizon 2020 programme under the SINE2020 project Grant No 654000.'''
+("This work benefited from the use of the SasView application, originally"
+"developed under NSF Award DMR-0520547. SasView also contains code developed"
+"with funding from the EU Horizon 2020 programme under the SINE2020 project"
+"Grant No 654000.")
 _acknowledgement_citation = \
-'''M. Doucet et al. SasView Version 4.2, Zenodo, 10.5281/zenodo.1412041'''
+'''M. Doucet et al. SasView Version %s, %s''' % (__version__, __DOI__)
 _acknowledgement =  \
-'''This work was originally developed as part of the DANSE project funded by the US NSF under Award DMR-0520547,\n but is currently maintained by a collaboration between UTK, UMD, NIST, ORNL, ISIS, ESS, ILL, ANSTO, TU Delft, DLS, BAM and the scattering community.\n\n SasView also contains code developed with funding from the EU Horizon 2020 programme under the SINE2020 project (Grant No 654000).\nA list of individual contributors can be found at: http://www.sasview.org/contact.html
-'''
+("This work was originally developed as part of the DANSE project funded by"
+"the US NSF under Award DMR-0520547,\n but is currently maintained by a"
+"collaboration between UTK, UMD, NIST, ORNL, ISIS, ESS, ILL, ANSTO, TU Delft,"
```

Comment:
Contributors appear multiple times. Separating them out could be useful, but might not be straight forward.

Reply:
agreed .... on both points at this time. I would like to think a bit about what the right answer is on this one I think?